push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/NotificationRouter.java

```diff
@@ -98,10 +98,17 @@ public void submit(PushApplication pushApplication, InternalUnifiedPushMessage m
             variants.addAll(pushApplication.getVariants());
         }
 
+        // TODO: Not sure the transformation should be done here...
```

Comment:
we might need a JIRA to cover this TODO, to find a proper place for this transition. One that is easier to test